content/docs/introduction/usage-metrics.md

```diff
@@ -4,33 +4,55 @@ enableTableOfContents: true
 updatedOn: '2024-02-26T19:37:28.835Z'
 ---
 
-This topic describes [Storage](#storage), [Compute](#compute), and [Project](#projects) usage metrics in detail so that you can better manage your [plan](/docs/introduction/plans) allowances and extra usage. 
+This topic describes [Storage](#storage), [Compute](#compute), and [Project](#projects) usage metrics in detail so that you can better manage your [plan](/docs/introduction/plans) allowances and extra usage.
 
 ## Storage
 
-_Storage_ is the total volume of data and history stored in Neon, measured in gibibytes (GiB). It includes the following:
+In Neon, your storage is made up of the combined total of these two elements:
 
-- **Data size**
+- **Data Size**
 
-  The size of all databases in your Neon projects. You can think of this as a _snapshot_ of your data at a point in time.
+    This is the current amount of data stored in your databases. You can think of this as a _snapshot_ of your database's size at any given moment.
 
 - **History**
 
-  Neon retains a history of changes for all branches to support _point-in-time restore_ and _branching_.
+    Your history consists of Write-Ahead Log (WAL) records, which log all changes made to your database over time. This shared history is what enables features like instant [branching](/docs/introduction/branching) and [point-in-time restore](/docs/introduction/point-in-time-restore). Initially, branches don’t add extra storage since they use shared data from existing snapshots.
```

Comment:
What do you think about leaving out mention of "shared" here? It's related to branching, which we don't mention yet. Also, what about mentioning: The amount of history you retain is configurable. See [Tips for minimizing storage](#tips-for-minimizing-storage).